### ISSUE TITLE
Deprecation warnings building with dmd 2.075

### ISSUE THREAD

**Issue by andresclari** [mysql-d/mysql-native#125]
After upgrading to DMD 2.075 the following deprecation notices were spit by compiler, referring to:

_Deprecation: function std.string.munch!(string, string).munch_

According to the docs on std.string, an alternative from std.algorithm should be used instead.

```
mysql-native-1.1.0/mysql-native/source/mysql/protocol/packet_helpers.d(68,7): Deprecation: function std.string.munch!(string, string).munch is deprecated - This function is obsolete and will be removed May 2018. See the docs for more details
mysql-native-1.1.0/mysql-native/source/mysql/protocol/packet_helpers.d(70,7): Deprecation: function std.string.munch!(string, string).munch is deprecated - This function is obsolete and will be removed May 2018. See the docs for more details
mysql-native-1.1.0/mysql-native/source/mysql/protocol/packet_helpers.d(112,7): Deprecation: function std.string.munch!(string, string).munch is deprecated - This function is obsolete and will be removed May 2018. See the docs for more details
mysql-native-1.1.0/mysql-native/source/mysql/protocol/packet_helpers.d(114,7): Deprecation: function std.string.munch!(string, string).munch is deprecated - This function is obsolete and will be removed May 2018. See the docs for more details
mysql-native-1.1.0/mysql-native/source/mysql/protocol/packet_helpers.d(181,7): Deprecation: function std.string.munch!(string, string).munch is deprecated - This function is obsolete and will be removed May 2018. See the docs for more details
mysql-native-1.1.0/mysql-native/source/mysql/protocol/packet_helpers.d(183,7): Deprecation: function std.string.munch!(string, string).munch is deprecated - This function is obsolete and will be removed May 2018. See the docs for more details
mysql-native-1.1.0/mysql-native/source/mysql/protocol/packet_helpers.d(264,7): Deprecation: function std.string.munch!(string, string).munch is deprecated - This function is obsolete and will be removed May 2018. See the docs for more details
mysql-native-1.1.0/mysql-native/source/mysql/protocol/packet_helpers.d(266,7): Deprecation: function std.string.munch!(string, string).munch is deprecated - This function is obsolete and will be removed May 2018. See the docs for more details
mysql-native-1.1.0/mysql-native/source/mysql/protocol/packet_helpers.d(268,7): Deprecation: function std.string.munch!(string, string).munch is deprecated - This function is obsolete and will be removed May 2018. See the docs for more details
mysql-native-1.1.0/mysql-native/source/mysql/protocol/packet_helpers.d(270,7): Deprecation: function std.string.munch!(string, string).munch is deprecated - This function is obsolete and will be removed May 2018. See the docs for more details
mysql-native-1.1.0/mysql-native/source/mysql/protocol/packet_helpers.d(272,7): Deprecation: function std.string.munch!(string, string).munch is deprecated - This function is obsolete and will be removed May 2018. See the docs for more details
```

**Comment by Abscissa** [comment]
Being addressed in PR #124

**Comment by Abscissa** [comment]
Fix merged into master.
